tensorboard/components/vz_line_chart2/vz-line-chart-tooltip.ts

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+namespace vz_line_chart2 {
+
+Polymer({
+  is: 'vz-line-chart-tooltip',
```

Comment:
As discussed, some comment about how this (logic-less) component is used (e.g. it's designed to be instantiated by vz-chart-tooltip, and populated by logic from vz-line-chart) would be helpful.